app/pages/index.vue

```diff
@@ -107,21 +107,19 @@ const storiesFiltered = computed(() => {
             Tentang Kami
           </h2>
           <p class="tracking-wide text-gray-600">
-            Hydra is a Y Combinator-backed data processing company with a
-            globally remote team. We are passionate PostgreSQL and backend
-            engineers from Microsoft Azure, Heroku, and Citus Data. We set out
-            to build the best all-in-one database for app development. We
-            believe DuckDB is a remarkable technology made revolutionary within
-            Postgres.<br />
+            Kami hadir untuk memenuhi visi yaitu menjadikan literasi terutama
+            membaca buku menjadi kebiasaan baru di masyarakat khususnya di
+            Yogyakarta. Yogyakarta Book Party berdiri pada tahun 2023 hingga
+            saat ini memiliki lebih dari 800+ anggota dengan 4 organisasi yang
+            berjalan didalamnya.<br />
```

Comment:
Kami hadir untuk memenuhi visi, yaitu menjadikan literasi, terutama membaca buku, menjadi kebiasaan baru di masyarakat, khususnya di Yogyakarta. Yogyakarta Book Party berdiri pada tahun 2023 hingga saat ini memiliki lebih dari 800 anggota dengan 4 organisasi yang berjalan di dalamnya.
```suggestion
            Kami hadir untuk memenuhi visi, yaitu menjadikan literasi, terutama membaca buku, menjadi kebiasaan baru di masyarakat, khususnya di Yogyakarta. Yogyakarta Book Party berdiri pada tahun 2023 hingga saat ini memiliki lebih dari 800+ anggota dengan 4 organisasi yang berjalan di dalamnya.<br />
```